src/sidebar/media-embedder.js

```diff
@@ -25,11 +25,47 @@ function iframe(src) {
   return iframe_;
 }
 
+/**
+ * Return a formatted YouTube URL query string containing allowed params
+ * e.g. '?t=5'
```

Comment:
`?t=5` could occur in the input to this function but won't occur in the output here. Perhaps it would be clearer to give an example input & output in this JSDoc comment?